File: Arrays/min_max.cpp
// Minimum and Maximum Element in Aray
#include <iostream>
#include <limits.h>
using namespace std;

int main()
{
    int mini = INT_MAX;
    int max = INT_MIN;

    int size = 6;
    int arr[] = {100, -4, 3, 20, 6, 5};

    for (int i = 0; i < size; i++)
    {
        if (arr[i] < mini)
        {
            mini = arr[i];
        }
        if (arr[i] > max)
        {
            max = arr[i];
        }
    }
    cout << "Maximum Number in Array is : " << max << endl;
    cout << "Minimum Number in Array is : " << mini << endl;
}

// if 100 > -1000:

File: Arrays/MinMax.py
arr = [3, 5, 4, 1, 9]

mini = 100
maxi = 0

print(arr)

for i in range(len(arr)):
    if arr[i] < mini:
        mini = arr[i]
    if arr[i] > maxi:
        maxi = arr[i]
        
print("Minimum number is : ",mini)
print("Maximum number is : ",maxi)